packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md

```diff
@@ -4,206 +4,172 @@ excerpt: Upgrade SAML Apps to SHA256
+* Roll a SAML 2.0 app integration back to its original SHA1 certificate
 
 **What you need**
 
-* A SAML app to upgrade. See [Building a SAML SSO integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration).
+A SAML app to upgrade. See [Building a SAML SSO integration](/docs/guides/build-sso-integration/saml2/main/#prepare-your-integration).
 
 ---
 
-## About SAML apps and SHA256 certificates
+Okta recommends that you upgrade SAML 2.0 app integrations that use SHA1 certificates to use SHA256 certificates instead. SHA256 is a more secure cryptographic hash function that superseded SHA1 in 2002. If your ISV doesn't accept certificates with an SHA256 signature, you can continue to use the previous SHA1 certificate. This guide also covers how to revert your app integration back to its original SHA1 certificate if there’s an issue.
 
-Certificates with a SHA256 signature are supported for SAML 2.0 applications with Okta. You can create new integrations that use SHA256 certificates and update existing integrations from SHA1 certificates to SHA256 certificates. Existing integrations are not changed automatically. The SHA256 certificates and the SHA1 certificates are self-signed.
+You can upgrade and revert certificates in the Admin Console, and also programmatically using the [Applications API](/docs/reference/api/apps/). This guide covers both options.
 
-### Why should I do this?
+> **Note:**  New SAML 2.0 app integrations automatically use SHA256 certificates. Those created with this guide are self-signed.
 
-To take advantage of the additional security features of SHA256 certificates.
+## Upgrade and revert apps with the Admin Console
 
-### New SAML 2.0 app integrations
+After you've created your SAML 2.0 app integration, you can use the Admin Console to check the type of certificate it currently uses.
 
-New SAML 2.0 app integrations automatically use SHA256 certificates.
+1. Open the Admin Console for your org.
+1. Go to **Applications** > **Applications** to view the current app integrations.
+1. Select your app integration.
+1. Go to the **SAML Signing Certificates** section of the **Sign On** tab.
 
-As instructed below, upload the SHA256 certificate to the ISV.
+   <div class="three-quarter border">
 
-### Existing SAML 2.0 app integrations
+      ![The SAML Signing Certificates section of the Applications UI](/img/updating-saml-certs/signing-certificates-ui.png)
 
-To update existing app integrations, you first need to list your apps and get the app id, name, and label for each app to update. Then for each app to update, perform the following steps:
+   </div>
 
-1. Generate a new application key credential.
-1. Update the key credential for the app to specify the new signing key id.
-1. Upload the new certificate to the ISV (this step cannot be automated.)
+1. If a certificate of **Type** SHA-2 is active, you don’t need to upgrade the certificate.
```

Comment:
Should these two items be in an unordered list beneath the action step preceding them?
```suggestion
* If a certificate of **Type** SHA-2 is active, you don’t need to upgrade the certificate.
```